src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] items = new Task[100];
```

Comment:
It would be good to convert the 100 into a constant, make it more meaningful instead of just having a magic number